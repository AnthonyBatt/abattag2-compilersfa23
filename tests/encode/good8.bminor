"I'm writing sentences that will hopefully test if regular english does things properly.\n\tI like the numbers 7, 4, and 0; that was a semicolon. :)\n"
