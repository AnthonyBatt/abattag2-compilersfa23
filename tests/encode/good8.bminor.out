
For the purpose of being able to see tabs, spaces, etc. printed I'm delimitng the start and end of the printed strings with ><

decoded string: >I'm writing sentences that will hopefully test if regular english does things properly.
	I like the numbers 7, 4, and 0; that was a semicolon. :)
<
encoded string >"I\'m writing sentences that will hopefully test if regular english does things properly.\n\tI like the numbers 7, 4, and 0; that was a semicolon. :)\n"<
